quarkus/server/build.gradle.kts

```diff
@@ -91,7 +91,7 @@ distributions {
       from(project.layout.buildDirectory.dir("quarkus-app"))
       from("distribution/NOTICE")
       from("distribution/LICENSE")
-      exclude("lib/main/io.quarkus.quarkus-container-image*")
+      from("README.md")
```

Comment:
The current README isn't really helpful when it was shipped with binary package. The binary distribution package will need a readme to describe how it runs instead of how it builds

Reply:
I added quick `README.md` in the distributions. That said, the documentation should be updated imho (I plan to do that in another PR).